.circleci/config.yml

```diff
@@ -668,12 +668,9 @@ jobs:
       - run:
           name: Build development contracts
           command: |
-            for contract_dir in ./contracts/*/; do
-              name=$(basename $contract_dir)
-              echo "Building $name ..."
-              docker run --volumes-from with_code cosmwasm/rust-optimizer:0.10.5 ./contracts/$name
-              docker cp with_code:/code/artifacts/$name.wasm ./artifacts/$name.wasm
-            done
+            echo "Building all contracts under ./contracts"
+            docker run --volumes-from with_code cosmwasm/rust-optimizer:0.10.6 ./contracts/*/
```

Comment:
Note: The glob `./contracts/*/` is supposed to be expanded in the host shell, right? This works as long as the expanded paths remain relative, but if the outer shell expands those to absolute paths and passes them to the guest, this breaks.

Reply:
Yes, glob expansion happens even before command processing, so, this is expandend in the host. And yes, this works because these paths are not only relative, but also relatively matching; that is, the directory structure in the container (relatively) matches the directory structure in the host. Relatively here means, wrt the working directory on both sides.